modules/osdk-updating-131-to-1361.adoc

```diff
@@ -0,0 +1,279 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-updating-projects.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-updating-projects.adoc
+// * operators/operator_sdk/helm/osdk-helm-updating-projects.adoc
+// * operators/operator_sdk/helm/osdk-hybrid-helm-updating-projects.adoc
+// * operators/operator_sdk/java/osdk-java-updating-projects.adoc
+
+ifeval::["{context}" == "osdk-golang-updating-projects"]
+:golang:
+:type: Go
+endif::[]
+ifeval::["{context}" == "osdk-ansible-updating-projects"]
+:ansible:
+:type: Ansible
+endif::[]
+ifeval::["{context}" == "osdk-helm-updating-projects"]
+:helm:
+:type: Helm
+endif::[]
+ifeval::["{context}" == "osdk-hybrid-helm-updating-projects"]
+:hybrid:
+:type: Hybrid Helm
+endif::[]
+ifeval::["{context}" == "osdk-java-updating-projects"]
+:java:
+:type: Java
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="osdk-upgrading-projects_{context}"]
+= Updating {type}-based Operator projects for Operator SDK {osdk_ver}
+
+The following procedure updates an existing {type}-based Operator project for compatibility with {osdk_ver}.
+
+.Prerequisites
+
+* Operator SDK {osdk_ver} installed
+* An Operator project created or maintained with Operator SDK {osdk_ver_n1}
+
+.Procedure
+
+// The following few steps should be retained/updated for each new migration procedure, as they're just bumping the OSDK version for each language type.
+
+. Edit your Operator project's Makefile to update the Operator SDK version to {osdk_ver}, as shown in the following example:
++
+.Example Makefile
+[source,make,subs="attributes+"]
+----
+# Set the Operator SDK version to use. By default, what is installed on the system is used.
+# This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
+OPERATOR_SDK_VERSION ?= v{osdk_ver} <1>
+----
+<1> Change the version from `{osdk_ver_n1}` to `{osdk_ver}`.
+ifdef::helm[]
+. Edit your Operator's Dockerfile to update the Helm Operator version to {osdk_ver}, as shown in the following example:
++
+.Example Dockerfile
+[source,docker,subs="attributes+"]
+----
+FROM registry.redhat.io/openshift4/ose-helm-rhel9-operator:v{osdk_ver} <1>
+----
+<1> Update the Helm Operator version from `{osdk_ver_n1}` to `{osdk_ver}`.
+endif::[]
+ifdef::ansible[]
+. Edit your Operator's Dockerfile to update the `ansible-operator` base image version to {osdk_ver}, as shown in the following example:
++
+.Example Dockerfile
+[source,docker,subs="attributes+"]
+----
+FROM registry.redhat.io/openshift4/ose-ansible-operator:v{osdk_ver}
```

Comment:
@adellape Same here as well. Instead of {osdk_ver} it should be {product-version}, so that we get `registry.redhat.io/openshift4/ose-ansible-operator:v4.17`.

Reply:
https://github.com/openshift/openshift-docs/pull/82521